frontend/client/src/screens/Settings.js

```diff
@@ -241,7 +215,6 @@ const SettingsBase = observer((props) => {
                 required
                 disabled={true}
                 aria-required="true"
-                className={input.root}
                 style={{ backgroundColor: '#e0e0e0' }}
```

Comment:
There are still quite a few lines like this around (style=[something]) that I think can also be moved to scss files. Especially ones like line 158 of this file where lots of styling is done inline.

Reply:
https://github.com/covidwatchorg/portal/issues/449